python/cugraph/cugraph/structure/property_graph.py

```diff
@@ -898,12 +1070,32 @@ def select_vertices(self, expr, from_previous_selection=None):
 
         Returns
         -------
-        PropertySelection instance to be used for calls to extract_subgraph()
-        in order to construct a Graph containing only specific vertices.
+        PropertySelection
+            used for calls to extract_subgraph()
+            in order to construct a Graph containing only specific vertices.
 
         Examples
         --------
-        >>>
+        >>> import cugraph
+        >>> import cudf
+        >>> from cugraph.experimental import PropertyGraph
+        >>> df = cudf.DataFrame(columns=["src", "dst", "some_property"],
+        ...                                          data=[(99, 22, "a"),
+        ...                                                (98, 34, "b"),
+        ...                                                (97, 56, "c"),
+        ...                                                (96, 88, "d"),
+        ...                                              ])
+        >>> pG = PropertyGraph()
+        >>> pG.add_edge_data(df, type_name="etype", vertex_col_names=("src", "dst"))
+        >>> vert_df = cudf.DataFrame({"vert_id": [99, 22, 98, 34, 97, 56, 96, 88],
+        ...                                       "v_prop": [1 ,2 ,3, 4, 5, 6, 7, 8]})
+        >>> pG.add_vertex_data(vert_df, type_name="vtype", vertex_col_name="vert_id")
+        >>> selection = pG.select_vertices("(_TYPE_ == 'vtype') & (v_prop > 4)")
+        >>> G = pG.extract_subgraph(selection=selection,
+        ...                         create_using=cugraph.Graph(directed=True),
+        ...                         renumber_graph=True)
```

Comment:
I would leave these args out since they shouldn't influence this example and their presence could make it seem as if they're required.